jcache-1.1/times-table/pom.xml

```diff
@@ -137,7 +137,7 @@
                 <activeByDefault>true</activeByDefault>
             </activation>
             <properties>
-                <profile.hazelcast.version>3.9.3</profile.hazelcast.version>
+                <profile.hazelcast.version>3.12.6</profile.hazelcast.version>
```

Comment:
Because we have a profile using JSR 1.0.0 and HZ 3.9, I'm not sure that we can upgrade the code and XML to 4.0. On the other hand, maybe we can freely remove the code sample for JSR 1.0.0. WDYT @vbekiaris ?

Reply:
This code sample demonstrates a subtle difference in how you obtain caches in JCache 1.1 vs 1.0 (associated blog post https://hazelcast.com/blog/times-table-jcache-1-1/), so I wouldn't remove the `jcache10` profile (yet). I think keeping both profiles running on 3.y is best for as long as we keep this code sample.